tv_apps/tv_shared/js/scrollable.js

```diff
@@ -93,6 +96,52 @@
       return this.frameElem.getBoundingClientRect();
     },
 
+    getScrollDistance: function(itemElem) {
+      return this._getScrollOffset(itemElem);
+    },
+
+    endSlide: function() {
+      // remove transition we added
+      this.listElem.classList.add('no-transition');
+      getComputedStyle(this.listElem).width;
```

Comment:
It's not an usual behavior to force reflow like this way. If you have to do this, please add some comment to explain why.